Datastore/src/Query/AggregationQueryResult.php

```diff
@@ -162,12 +174,46 @@ public function getTransaction()
     }
 
     /**
-     * Gets the timsstamp when Aggregation was executed.
+     * Gets the timestamp when Aggregation was executed.
      *
      * @return Timestamp
      */
     public function getReadTime()
     {
         return $this->readTime;
     }
+
+    /**
+     * Gets the ExplainMetrics object if the explainOptions parameter was set for
+     * the query
+     *
+     * @return null|ExplainMetrics
+     */
+    public function getExplainMetrics(): null|ExplainMetrics
+    {
+        return $this->explainMetrics;
+    }
+
+    private function fixDurationFormat(array $metrics): array
+    {
+        // The current protobuf library does not support the current json representation
+        // of the well-known type Duration.
+        // Hence we have to convert the object format into a string format for the merging from json to work.
+        // If the protobuf library gets updated, this should be removed.
+        if (!isset($metrics['executionStats']) && !isset($metrics['executionStats']['executionDuration'])) {
+            return $metrics;
+        }
+
+        // The REST version returns the executionDuration in a String format. If is a string should be ready to go
+        if (isset($metrics['executionStats']) && is_string($metrics['executionStats']['executionDuration'])) {
+            return $metrics;
+        }
+
+        $seconds = $metrics['executionStats']['executionDuration']['seconds'];
```

Comment:
I'm a little surprised that this is necessary, since we're saying special conversion is needed for gRPC (which uses protobuf messages) in order to convert back into a protobuf message, but NOT with REST, which doesn't use protobuf messages...